lib/mixins/form.js

```diff
@@ -1,17 +1,34 @@
 export default {
+    computed: {
+        inputClass() {
+            return [
+                this.size ? `form-control-${this.size}` : null,
+                this.state ? `form-control-${this.state}` : null
+            ];
+        },
+        custom() {
```

Comment:
Hmm, this shouldn't  be in here anymore... s plain was moved only to the form-* components that have that option.

Same for the inputClass() computed prop (doesn't apply to all form components)